src/main/java/server/ProxyHandler.java

```diff
@@ -0,0 +1,30 @@
+package main.java.server;
+
+import main.java.server.request.Request;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class ProxyHandler {
+
+    public static String sendRequest(Request request) throws IOException {
+        String path = request.getRequestPath().substring(1);
+
+        URL url = new URL(path);
+        HttpURLConnection con = (HttpURLConnection) url.openConnection();
+
+        BufferedReader in = new BufferedReader(
+                new InputStreamReader(con.getInputStream()));
+        String inputLine;
+        StringBuffer content = new StringBuffer();
+        while ((inputLine = in.readLine()) != null) {
+            content.append(inputLine);
+        }
+
+        in.close();
+        return content.toString();
```

Comment:
This is probably hard to test, but also probably something where it's good to think about how we can test it. Feels like a place where a refactor could break things without alarm bells going off